packages/swingset-liveslots/src/liveslots.js

```diff
@@ -790,7 +790,11 @@ function build(
     let result;
     if (virtual || durable) {
       assert.equal(type, 'object');
-      val = vrm.reanimate(baseRef);
+      try {
+        val = vrm.reanimate(baseRef);
+      } catch (err) {
+        Fail`failed to reanimate ${iface} (${slot}); check deserialize preceeding kind definitions`;
```

Comment:
should `err.message` be included in the `Fail` message?

(where is `err` used? looks like nowhere; how does that pass lint?)

Reply:
huh, yeah. `err` is unused. maybe it's a special exception in the linter.

the `err.message` is from `reanimate` and that level of detail I don't think is pertinent to percolate: https://github.com/Agoric/agoric-sdk/blob/621a889bd62a6d42d5bde732cee37816ea8925bb/packages/swingset-liveslots/src/virtualReferences.js#L312-L313

If someone needs to debug why the reinimate failure is happening I expect they'll invoke a debugging or logging